api/v1alpha1/ocivalidator_types.go

```diff
@@ -52,17 +47,36 @@ type OciRegistryRule struct {
 }
 
 func (r OciRegistryRule) Name() string {
-	return fmt.Sprintf("%s/%s", r.Host, r.Artifacts)
+	var artifacts strings.Builder
+	artifacts.WriteString("[")
+	l := len(r.Artifacts)
+	for i, a := range r.Artifacts {
+		artifacts.WriteString(a.Ref)
+		if i < l-1 {
+			artifacts.WriteString(", ")
+		}
+	}
+	artifacts.WriteString("]")
+
+	return fmt.Sprintf("%s/%s", r.Host, artifacts.String())
 }
 
-type Auth struct {
-	// Username is the username to use for authentication.
-	// This may also be an AWS_ACCESS_KEY_ID.
-	Username string `json:"username" yaml:"username"`
+type Artifact struct {
+	// Ref is the path to the artifact in the host registry that should be validated.
+	// An individual artifact can take any of the following forms:
+	// <repository-path>/<artifact-name>
+	// <repository-path>/<artifact-name>:<tag>
+	// <repository-path>/<artifact-name>@<digest>
+	//
+	// When no tag or digest are specified, the default tag "latest" is used.
+	Ref string `json:"ref" yaml:"ref"`
+
+	// Download specifies whether a download attempt should be made for the artifact
+	Download bool `json:"download" yaml:"download"`
```

Comment:
For convenience please add `omitempty` that way the OCIValidator spec can be a lot cleaner. Users will only need to specify `download: true` for any artifact(s) they want to download and can omit it otherwise.

Reply:
sure thing. the main reason i didnt opt for that initially was that it felt like defaulting to not downloading was the wrong move. im ok with making the change though